test/rest_api/test_hinted_handoff.py

```diff
@@ -0,0 +1,21 @@
+import requests
+import urllib.parse
+import base64
+
+def test_hinted_handoff_sync_point(rest_api):
+    resp = rest_api.send('POST', "hinted_handoff/sync_point")
+    sync_point = resp.json()
+    sync_point_b = base64.b64decode(sync_point.encode('ascii'))
```

Comment:
I think that you should add a comment here in order to explain why we need to partially decode the sync point.

Reply:
I think the new explanation is good enough.